staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -328,3 +353,28 @@ func (v *specStandardValidatorV3) validate(schema *apiextensions.JSONSchemaProps
 
 	return allErrs
 }
+
+// ValidateCustomResourceDefinitionSubResources statically validates
+func ValidateCustomResourceDefinitionSubResources(subResources *apiextensions.CustomResourceSubResources, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if subResources == nil {
+		return allErrs
+	}
+
+	if subResources.Scale != nil {
+		if len(subResources.Scale.SpecReplicasPath) == 0 {
```

Comment:
need to ensure this is a valid constrained jsonpath

Reply:
Done.